docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py

```diff
@@ -0,0 +1,68 @@
+from mkdocs.plugins import BasePlugin
```

Comment:
What is this script for and why is it in the docs directory?

Reply:
This script is a mkdocs plugin which allows .md files outside of the docs folder to be compiled and be part of the readthedocs documentation. Readthedocs has limitations, and it does not support reference to files outside of the doc root (see #6329). This plugin creates a temporary folder and copies all the .md files of the repository there to make links from the wiki to .md in the moodules directory possible.

Reply:
Does it need to go in the docs directory? What advantage does having a script have over simply moving the doc root to be the same as the LORIS root?

Reply:
Readthedocs throw an error if the doc root is changed to be the same as the LORIS root, beacuse this change makes it break. The only way to achieve this is with a plugin.

Reply:
@driusan Scripts have been moved from /docs to /readthedocs.